tests/test_z_cmdline.py

```diff
@@ -635,7 +635,11 @@ def _alwayscopy_not_supported():
     # see: https://github.com/pypa/virtualenv/issues/565
     if hasattr(platform, 'linux_distribution'):
         _dist = platform.linux_distribution(full_distribution_name=False)
-        if _dist[0] == 'centos' and _dist[1][0] == '7':
+        (name, version, arch) = _dist
+        if (
```

Comment:
this formatting looks akward, maybe instead use the ``any`` with two args, one per line :+1:

Reply:
Nice trick !